users/permissions.py

```diff
@@ -0,0 +1,15 @@
+"""
+Contains custom permissions
+"""
+from rest_framework.permissions import BasePermission
+
+
+class IsAdmin(BasePermission):
+    """
+    User must be super user or staff member to access the view
+    """
+
+    def has_permission(self, request, view):
+        return bool(
+            request.user and request.user.is_staff and request.user.is_superuser
```

Comment:
I think we want to use or instead of and since the user will be admin if he is staff or superuser
```suggestion
            (request.user and request.user.is_staff or request.user.is_superuser)
```